scripts/deprovision-tenant.sh

```
@@ -0,0 +1,130 @@
+  export CDK_PARAM_REG_API_GATEWAY_URL="NA"
+  export CDK_PARAM_EVENT_BUS_ARN=arn:aws:service:::resource
+  export CDK_PARAM_CONTROL_PLANE_SOURCE="NA"
+  export CDK_PARAM_ONBOARDING_DETAIL_TYPE="NA"
+  export CDK_PARAM_PROVISIONING_DETAIL_TYPE="NA"
+  export CDK_PARAM_PROVISIONING_EVENT_SOURCE="NA"
+  export CDK_PARAM_APPLICATION_NAME_PLANE_SOURCE="NA"
+  export CDK_PARAM_OFFBOARDING_DETAIL_TYPE="NA"
+  export CDK_PARAM_DEPROVISIONING_DETAIL_TYPE="NA"
+
+  echo "undeploying tenant template $STACK_NAME"
+  npx cdk destroy $STACK_NAME --force
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)